.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
     token: $SONARCLOUD_TOKEN
 
 script:
-  - ./gradlew check && chmod +x start_local_a+_env.sh && ./start_local_a+_env.sh && sonar-scanner
+  - ./start_local_a+_env.sh && ./gradlew check && chmod +x start_local_a+_env.sh && sonar-scanner
```

Comment:
Shouldn't `chmod +x start_local_a+_env.sh` come before `./start_local_a+_env.sh` ? Or is it unnecessary?

Reply:
this is now right as it should be - first the instance of A+ is started, and then we run all the tests because our "integration" test is run with the same `gradle check` (`test`) actually

Reply:
yes, now I see... and surprisingly this still works :rofl:

Reply:
Shouldn't the `chmod +x start_loca_a+_env.sh` be the first one (or removed completely)?

Reply:
Oops, I commented before refreshing the page and seeing your reply